src/components/Ecosystem/MemberList.tsx

```diff
@@ -106,7 +110,7 @@ const MemberList = () => {
 			});
 			const memberList =
 				data?.data?.data?.length > 0 &&
-				data?.data?.data?.map((member: IMemberList) => {
+				data?.data?.data?.map((member) => {
```

Comment:
any specific reason for removing the interface?

Reply:
Resolved